backend/schemas/source/data/cluster-names-20230626.csv

```diff
@@ -1,42 +1,41 @@
-ID,NAME,SORT_NUM
-55,HEAD START CLUSTER,179
-50,477 CLUSTER,90
-51,"MATERNAL, INFANT, AND EARLY CHILDHOOD HOME VISITING CLUSTER",207
-1,N/A,1
-2,RESEARCH AND DEVELOPMENT,10
-3,STUDENT FINANCIAL ASSISTANCE,20
-4,STATE CLUSTER,30

```

Comment:
`STATE CLUSTER` was removed in favor of a const in the libsonnet. Is that the right decision? If so, we should also remove `N/A` from line 5.

Reply:
I see what's happening. There are three conditions for "cluster name" in the Federal Awards awards:

1. One of the listed cluster names, which includes "N/A"
2. State cluster, which requires an entry in "state_cluster_name"
3. Other, which requires an entry in "other_cluster_name"

`STATE_CLUSTER` was removed from the source JSON so it doesn't show up in the options for condition `1`.  `N/A` remains because it's included in condition `1`. 

Given that, I think we should remove `N/A` from this file.

Reply:
That makes sense. It probably made it by because of the conversion of N/A to NAN? Either way, affirming the logic here.